QtCollider/style/routines.hpp

```diff
@@ -160,7 +160,7 @@ void drawSunken(QPainter* p, const QPalette& plt, const ShapeT& shape, const QCo
     // shadow
 
     QColor c2 = color;
-    c2.setAlpha(150);
+    c2.setAlpha(color.alpha() / 2);
```

Comment:
better: `c2.setAlpha(color.alpha * 0.5);`

Reply:
right, fixed! thanks

Reply:
Why though is `color.alpha * 0.5` *better* than `color.alpha() / 2`? From a computing perspective `/2` is a cheap operation and therefore commonly used from low- to high-level, and also the common mathematical notation would be `/2`.

In the end I am fine with either way, but I think it is hard to argue and reason about this.